### ISSUE TITLE
Text composables: Android Basics with Compose

### ISSUE THREAD

**Issue by pjl87** [google-developer-training/basic-android-kotlin-compose-birthday-card-app#801]
**URL of codelab**
https://developer.android.com/codelabs/basic-android-kotlin-compose-text-composables?continue=https%3A%2F%2Fdeveloper.android.com%2Fcourses%2Fpathways%2Fandroid-basics-compose-unit-1-pathway-3%23codelab-https%3A%2F%2Fdeveloper.android.com%2Fcodelabs%2Fbasic-android-kotlin-compose-text-composables#4

**In which task and step of the codelab can this issue be found?**
STEP
5. Add a new text element

**Describe the problem**

In the code lab, the sample code shows a "surface" variable as follows:
class MainActivity : ComponentActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContent {
            HappyBirthdayTheme {
                // A surface container using the 'background' color from the theme
//------->       Surface(
                    modifier = Modifier.fillMaxSize(),
                    color = MaterialTheme.colorScheme.background
-----------------------------------------------------------------------------------------------------------------
HOWEVER, When the app is built using Android Studio Koala 2024.1.1, it doesn't have a "Surface" variable. Instead it uses a "Scaffold" variable in place of the "Surface" variable as follows:

class MainActivity : ComponentActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContent {
            HappyBirthdayTheme {
                // A surface container using the 'background' color from the theme       
                    Scaffold(modifier = Modifier.fillMaxSize()) {  innerPadding ->
                    GreetingText(
                        name = "Android",
                        modifier = Modifier.padding(innerPadding)
                        )
                     }
               }
        }
}
-----------------------------------------------------------------------------------------------------
In conclusion, when the code lab asks the student to change text styles and other elements of the @Composable it creates many issues with imports, code errors, and other elements in the code lab
                    

**Steps to reproduce?**
1. Go to...
2. Click on...
3. See error...

**Versions**
_Android Studio version:_  Android Studio Koala | 2024.1.1
Build #AI-241.15989.150.2411.11948838, built on June 10, 2024
Runtime version: 17.0.10+0-17.0.10b1087.21-11609105 amd64

_API version of the emulator:_ 
29

**Additional information**
_Include screenshots if they would be useful in clarifying the problem._
